src/lib/zf/attr.c

```diff
@@ -316,7 +316,9 @@ int zf_attr_get_str(struct zf_attr* attr, const char* name, char** val)
   switch( f->type ) {
   case zf_attr_type_str: {
     char** p = get_field_str(attr, f);
-    if( *p != NULL ) {
+    if( strcmp(f->name, "log_file")==0 )
+      *val = strdup(zf_log_file_name);
```

Comment:
should consider what happens if this strdup fails

Reply:
indeed, I've missed this one. Thanks.